pootle/core/initdb.py

```diff
@@ -295,3 +306,56 @@ def create_default_languages(self):
                 self._create_object(Language, **criteria)
             except:
                 pass
+
+    def create_default_languages_cldr(self):
+        """
+        Generate Language objects from CLDR database
+
+        You can optionally set up a list of 'fallback aliases' (mappings from language definitions existing in CLDR
+        to locale names which you need in your Pootle installation)
+        """
+        from pootle_language.models import Language
+        for attrs in self.get_cldr_languages():
+            code = attrs.pop('code')
+            self._create_object(Language, code=code, defaults=attrs)
+
+
+    def update_languages_to_cldr(self):
```

Comment:
No, `initdb` is meant to be used once to initialize the database from scratch, not to tweak it afterwards. If you think that this code might be used somewhere else in the future keep it on a branch on your fork, but we are not going to land unused code.

Re adding this feature of converting the languages to CLDR, we would like to wait for input from @dwaynebailey.